cmd/scp03.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * (C) Copyright 2021, Foundries.IO
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
```

Comment:
minor: SPDX-License-Identifier before copyright:

```
// SPDX-License-Identifier: GPL-2.0+                                            
/*                                                                              
 * (C) Copyright ...                                      
 */         
```

Reply:
um ok, uboot seems to have a mismatch of these all over...